### PR TITLE
Update to work with new frost-icons w/ icon packs

### PR DESCRIPTION
**Pull request by job13er**
This #major# change allows the component to work with the latest `ember-frost-core` which requires `frost-icon` to include a `pack` property (leaving that blank doesn't seem to actually work).
# CHANGELOG

 **Added** support for latest `ember-frost-core` (icon packs in `frost-icon`)
- **Changed** the `logoPath` and `stripPath` properties to `logoIcon` and `stripIcon` respectively
- **Added** an `iconPack` property which allows setting the `pack` property on internal `frost-icon` components used in the dialog.
- **Fixed** an issue where the class `frost-about-dialog` would be added to **all** `ember-remodel` modals when the about dialog launches.

### CODE REVIEW

File: addon/styles/addon.scss [hunk 1]
```diff
@@ -18,113 +18,113 @@ $modal-bow-shadow: rgba(0, 0, 0, .25);
       padding: 0;
       background-color: $frost-color-white;
       box-shadow: 0 3px 8px $modal-bow-shadow;
+    }
+  }
 
-      .frost-button {
-        outline: none;
-        background: transparent;
```
Comment (job13er) [inline addon/styles/addon.scss:24]: This diff looks ugly, but I basically just pulled the majority of the styles out from a couple nesting levels to live just below the `.frost-about-dialog` selector.

**Comment by sandersky** [comment]
:+1:

[Approved with PullApprove](https://pullapprove.com/ciena-frost/ember-frost-about-dialog/pull-request/22/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ciena-frost/ember-frost-about-dialog)

File: addon/components/frost-about-dialog.js [hunk 2]
```diff
@@ -1,19 +1,63 @@
...
+
 import Ember from 'ember'
+import PropTypeMixin, {PropTypes} from 'ember-prop-types'
+
 import layout from '../templates/components/frost-about-dialog'
 
 const {Component, inject} = Ember
 
-export default Component.extend({
+export default Component.extend(PropTypeMixin, {
```
Comment (sglanzer-fire) [inline addon/components/frost-about-dialog.js:12]: Don't need the mixin anymore

Comment (job13er) [inline addon/components/frost-about-dialog.js:12]: Yeah, found that out after it was merged. I had checked Bunsen, which was still using the mixin too

**Pull request merged**